app/controllers/api/v1/exceptions_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V1
+    # lightweight wrapper to intercept all uncaught API-based exceptions and render JSON response
+    class ExceptionsController < ApiBaseController
+      def render_error
+        ::RequestUtils.log_exception(request, params, user: current_api_user, study: @study)
+        render json: ::RequestUtils.exception_json(request.env['action_dispatch.exception']),
```

Comment:
nice, let's just pass in the full request instead of the `request.env['action_dispatch.exception']`, so that the signature better matches the signature of the `log_exception` method